Misc/NEWS.d/next/C API/2018-08-04-00-59-44.bpo-34331.iaUkmU.rst

```diff
@@ -0,0 +1,2 @@
+Use singular/plural noun in error message when instantiating an abstract
+class with abstract method(s).
```

Comment:
An abstract class by definition has abstract methods; the last three words are tautological.

Reply:
So what’s the intented meaning here?
«an abstract class with non-overriden abstract methods» ?

Reply:
Done